### DISCUSSION TITLE
flet vs pynecone which to choose.

### DISCUSSION THREAD

**Discussion by saisaigraph** [reflex-dev/reflex#322] [General]
Please suggest, flet vs pynecone which to choose.
Thanks

**Comment by FHU-yezi** [comment]
Pynecone use React, and Flet use Flutter, this two packages are all the wrapper of other framework.

But Pynecone has smaller download size, which related to faster loading, so I prefer Pynecone.

**Comment by Alek99** [comment]
Pynecone is way better for web apps in my opinion. We make our whole website and docs in Pynecone. Flet cannot do this in there framework and there whole website is made in JavaScript. That sums up its limitations for building web apps

**Comment by kumaraguru1735** [comment]
> Please suggest, flet vs pynecone which to choose. Thanks

Use Flet to build android apps
Use Reflex to build Responsive Websites
